accepted/0000-prometheus-tls.md

```diff
@@ -0,0 +1,238 @@
+- Feature Name: Prometheus TLS support
+- Start Date: 2021-02-11
+- RFC PR:
+
+# Summary
+
+[summary]: #summary
+
+Secure connection to Prometheus server and exporters.
```

Comment:
Is the idea to provide optional encryption or mandatory?

Reply:
Updated.